client/state/test/index.js

```diff
@@ -0,0 +1,48 @@
+require( 'lib/react-test-env-setup' )();
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+describe( 'state', () => {
+	var createReduxStore;
+	before( () => {
+		createReduxStore = require( 'state' ).createReduxStore;
```

Comment:
It originally caused errors with import, since something up the chain was expecting window to be available. I can test it again as a simple import.